src/mame/layout/segaai.lay

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<mamelayout version="2">
```

Comment:
Missing the CC0 license grant for layouts.

Reply:
Added CC0 license to layout